### ISSUE TITLE
`Digital Signatures` is needed or thorium will be killed by group policy.

### ISSUE THREAD

**Issue by enihcam** [Alex313031/Thorium-Win#200]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when my thorium gets killed by group policy due to lack of Windows digital signatures.

**Describe the solution you'd like, including relevant patches or source**
Pack thorium binaries together with digital signatures.

**Additional Notes**
Take VSCode binary for example:
![image](https://github.com/Alex313031/Thorium-Win/assets/7010097/4c70e990-5e8f-4bc3-9859-b4e0da5ec2c8)

**Comment by Alex313031** [comment]
@gz83 @enihcam Yeah, in this unique case IDK what to say. Signing it is cost prohibitive, and this policy applies to all binaries.

**Comment by mpogue** [comment]
I have no experience with them, but [Certum](https://certum.store/data-safety/code-signing-certificates.html?as_dane_w_certyfikacie=5720) has open source code signing certificates for fairly reasonable prices. Their root certificate is trusted by Windows. I'd be willing to pitch in a few dollars towards it.

**Comment by gz83** [comment]
Since digital signature certificates are quite expensive, I am sorry to say that Thorium is currently unable to do this.
